src/GAP.jl

```diff
@@ -318,6 +320,28 @@ function randseed!(seed::Union{Integer,Nothing}=nothing)
     nothing
 end
 
+"""
+    versioninfo(io::IO = stdout; jll::Bool = false)
```

Comment:
Could you add `full` to the docstring as well?

Reply:
o.k.